### ISSUE TITLE
Support for Electron

### ISSUE THREAD

**Issue by Romiko** [NeuraLegion/cypress-har-generator#46]
Hi,
Please add support for Electron browser.

Thx

**Comment by kpittman-securus** [comment]
Please let me know if there is anything I can do to help get this implemented!

I just found out about this plugin, and we desperately need this during our CI runs (which are on Electron).

Even though Cypress has videos and screenshots, some of our setup code fails during `before` hooks, and the Cypress logs don't say why. Since we are doing API calls during this phase, there's nothing shown in the video or screenshots.

It would be absolutely IDEAL for us to be able to download a HAR and debug that way, but it would have to work in Electron.

**Comment by derevnjuk** [comment]
To make the plugin work with Electron, you just need to set the `--remote-debugging-port` switch while running Cypress tests as follows:

```bash
$ ELECTRON_EXTRA_LAUNCH_ARGS=--remote-debugging-port=9222 cypress run --browser electron
```

For details please refer to #151. Support for this switch has been implemented since version 5.11.0, which can be found in the release notes on our Github repository at https://github.com/NeuraLegion/cypress-har-generator/releases/tag/v5.11.0
